logic/src/commonMain/kotlin/com/wire/kalium/logic/data/notification/LocalNotification.kt

```diff
@@ -1,9 +1,21 @@
 package com.wire.kalium.logic.data.notification
 
 import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.feature.call.CallStatus
 
 /**
- * Kalium local data classes that contains all the necessary data for displaying Notifications,
+ * Kalium local data classes that contains all the necessary data for displaying Call Notification, or display call itself
+ * and suppose to be mapped (in platform side) into platform-specific objects to show the notification, or call
+ */
+data class LocalNotificationCall(
+    val conversationId: ConversationId,
+    val status: CallStatus,
+    val notificationTitle: String,
+    val notificationBody: String?
+)
```

Comment:
Does it need to be named `LocalNotification`?

What if we don't want to display notifications but just display the items in a list?

Maybe just `DetailedCall` or `CallDetails` is good enough and represents it better.

How the notifications will be displayed, what goes in the title, what goes in the body, or whatever, is responsibility of the app. So perhaps `Kalium` can leave it to Reloaded or other apps to figure out. Maybe provide some helpers as functions, but maybe not set in stone like this.


My suggestion:

```suggestion
data class LocalNotificationCall(
    val conversationId: ConversationId,
    val status: CallStatus,
    val notificationTitle: ConversationDetails
)
```

And do the mapping to a `CallNotification` on Reloaded. This way we let the consumers of Kalium use all the `ConversationDetails` data to display a notification.

Reply:
Why adding a new data class for notification ? Adding a new fields for `callerName` and `callMetaData` to `Call` data class isn't enough ?